pkg/asset/installconfig/gcp/client.go

```diff
@@ -233,6 +222,35 @@ func (c *Client) GetProjects(ctx context.Context) (map[string]string, error) {
 	return projects, nil
 }
 
+// GetZones uses the GCP Compute Service API to get a list of machine types from a project.
+func (c *Client) GetZones(ctx context.Context, project, filter string) (map[string]*compute.Zone, error) {
```

Comment:
There is some heavy overlap between this new function and the existing `AvailabilityZones` function. Can we avoid this duplication?

https://github.com/openshift/installer/blob/692bdd1255f7a3076b637f5bb77fdc4a0e71f12a/pkg/asset/machines/gcp/zones.go#L16

Reply:
@staebler I was thinking the same. I'm assuming we don't want to simply import the machines/gcp package here. However, I envision this being something in /pkg/client/gcp or similar so that it can be shared across assets (such as destroy). This feels more like an RFE or tech debt. Thoughts?

Reply:
Oh, yeah. I was thinking that the gcp client was in a more accessible package, not tucked away in the installconfig assets package. That is certainly beyond the scope of what this BZ fix should cover. Would you mind creating an Jira card that we can throw in the backlog to cover making the cloud clients more accessible?

Reply:
https://issues.redhat.com/browse/CORS-1598